queries.css

```diff
@@ -119,6 +118,51 @@
     border-left: 1px solid #f42;
     padding: 0 5px 0 5px;
   }
+
+  .categories-subtitle-img {
+    justify-content: space-between;
+  }
+
+  .categories-container {
+    display: grid;
+    grid-template-columns: 50% 50%;
+    grid-template-rows: repeat(5, 1fr);
+  }
+
+  .latest-news-container {
+    margin: 40px 20px 20px 20px;
+  }
+
+  .ln-grid {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(3, 1fr);
+  }
+
+  .ln-content {
+    flex-direction: column;
+  }
+
+  .ln-img img {
+    height: 120px;
+    width: 100%;
+  }
+
+  .ld-grid {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(3, 1fr);
+  }
+
+  .ld-content {
+    flex-direction: column;
+    margin-right: 10px;
+  }
+
+  .ld-img img {
+    height: 120px;
+    width: 100%;
+  }
```

Comment:
## Aim to Stay `DRY`
- [ ] The lines of code on line `#146` & `#162` appear to be similar and make you seem like you are repeating yourself. Would you mind merging them so as to not appear redundant?